frontend/public/components/dashboards-page/overview-dashboard/health-card.tsx

```diff
@@ -57,6 +59,11 @@ const getK8sHealthState = (openShiftFlag: boolean, k8sHealth: any): SubsystemHea
     : { message: `${getName(openShiftFlag)} ${ERROR}`, state: HealthState.ERROR };
 };
 
+const getAlerts = (alertsResults: any): Alert[] => {
```

Comment:
can we move this to common place so other component leveraging can use this? @rawagner @pcbailey 
```suggestion
const getAlerts = (alertsResults: any): Alert[] => {
const alertsResponse = alertsResults ? alertsResults.getIn([ALERTS_KEY, 'result'], []) : [];
  return Array.isArray(alertsResponse) ? alertsResponse : [];
};
```

Reply:
@cloudbehl Do you have any suggestions on a good location for it?

Reply:
i'd suggest `dashboard/health-card/utils.ts `

Reply:
I thought about that, but I wasn't sure if that was "common" enough.

Reply:
I have moved it to `dashboard/health-card/utils.ts`.